source/logrepl/cleaner.go

```diff
@@ -52,6 +53,14 @@ func Cleanup(ctx context.Context, c CleanupConfig) error {
 	var errs []error
 
 	if c.SlotName != "" {
+		// Terminate any outstanding backends which are consuming the slot before deleting it.
+		mrr := conn.Exec(ctx, fmt.Sprintf(
+			"SELECT pg_terminate_backend(active_pid) FROM pg_replication_slots WHERE slot_name='%s AND active=true'", c.SlotName,
```

Comment:
Hmm, aren't the quotes misplaced here?

```suggestion
			"SELECT pg_terminate_backend(active_pid) FROM pg_replication_slots WHERE slot_name='%s' AND active='true'", c.SlotName,
```

Reply:
Yes, but just on `slot_name` in pg SQL `true` does not need to be quoted.

Reply:
Fixed here f28285404c22064f4ac63af063fcc785358965c3